Update sessions.rst

ThomasLandauer opened the pull request:
First step to explain how to activate Namespaced Attributes, as requested by https://github.com/symfony/symfony-docs/issues/7378

ThomasLandauer committed:
Update sessions.rst

First step to explain how to activate Namespaced Attributes, as requested by https://github.com/symfony/symfony-docs/issues/7378

javiereguiluz committed:
Removed some repeated content and minor rewords

HeahDude commented on components/http_foundation/sessions.rst:
```diff
@@ -185,25 +170,27 @@ and remember me login settings or other user based state information.
 
 :method:`Symfony\\Component\\HttpFoundation\\Session\\Attribute\\AttributeBagInterface::remove`
-    Deletes an attribute by key.
+    Deletes an attribute by name.
```
and returns its value.

HeahDude reviewed (commented):
One minor comment, nice PR, thanks!

ThomasLandauer committed:
Update sessions.rst

As requested here :-) https://github.com/symfony/symfony-docs/pull/7643#pullrequestreview-29070751

HeahDude reviewed (approved):
👍

drupol commented:
Hi,

I would rather use a valid syntax example for `config.yml`:

```
services:
    session:
      class: Symfony\Component\HttpFoundation\Session\Session
      arguments: ["@session.storage", "@session.namespacedattributebag", "@session.flash_bag"]

    session.namespacedattributebag:
      class: Symfony\Component\HttpFoundation\Session\Attribute\NamespacedAttributeBag
```

ThomasLandauer commented:
Sorry, don't know what you mean - can't remember all details half a year later....

Do you want to *add* something, or do you want me to *change* something?

mhabibi commented on components/http_foundation/sessions.rst:
```diff
@@ -205,35 +219,22 @@ the array::
+        session:
+            class: Symfony\Component\HttpFoundation\Session\Session
+            arguments:
+                - @session.storage
```
shouldn't it be @session.storage.native?

javiereguiluz replied:
@HeahDude you approved this PR ... but I'd like to ask you what do you think about this comment from @mhabibi. Do you agree with it? Thanks!

HeahDude replied:
We don't care about the implementation here, so we don't need to be specific in that case and the alias is used. From Symfony 3.3, it should `SessionStorageInterface` instead (autowired by default). See https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/Resources/config/session.xml#L21, that keeps BC with the old alias.
We can merge as is, thanks!

javiereguiluz committed:
Reword and simplify

javiereguiluz commented:
@ThomasLandauer sorry it took us so long to merge this. I like a lot your improvements. I only removed some example that showed how to configure this on Symfony. Since this is the stand-alone component doc, we never show how to integrate it with the whole framework. Thanks!

pbowyer commented:
@javiereguiluz Can the examples that showed how to configure Symfony be added to the Symfony session documentation? It took ma a while to find this PR to confirm how to use `NamespacedAttributeBag` with Symfony 4.

javiereguiluz commented:
@pbowyer I've created #10494 to not forget about this. Thanks!
